README.md

```diff
@@ -56,59 +53,44 @@ See [Getting Started with Google Maps Platform](https://developers.google.com/ma
  _Note: Google Maps Platfom requires creation of a billing account, but [they offer](https://cloud.google.com/maps-platform/pricing/) $200 of free monthly usage, which is enough to support development._
 
 
-### Development & Fill the Database
-
-You will need to execute migration process, run
-```
-scripts/update
-```
-then reset database and repopulate with fixture data that including parse & geocode processing, run
-```
-scripts/reset_database
+### Kick-off & start local development
```

Comment:
With the new approach to setting up the project, we ended up with 350 seeded facilities after installation. However, with [the approach Roman suggested](https://spdfn.slack.com/archives/C04RGFZC1GW/p1707463361257669?thread_ts=1707462211.802249&cid=C04RGFZC1GW) previously, I consistently obtained 927 facilities. We have obtained 927 seeded facilities after the each initial installation for one year.